vars.sample

```diff
@@ -50,10 +50,20 @@ ORACLE_TABLESPACES_DIR=/home/oracle/app/oracle/oradata
 # }}}
 
 # Only needed for ETL
+ETL_SCRIPTS=/path/to/transmart-data/samples/postgres
+export PATH=$ETL_SCRIPTS:$PATH
```

Comment:
I agree, this addition is for the convenience of the tranSMART study data loader, when the study data traversal approach is used for uploading. This reliefs him/her from using the full path with the upload command (load_<datatype>.sh i.s.o. /path/to/transmart-data/samples/postgres/load_<datatype>.sh). Do you have other suggestions to make life of study data loaders easier.